docs/using-fake-timers.md

```diff
@@ -0,0 +1,49 @@
+---
+id: using-fake-timers
+title: Using Fake Timers
+sidebar_label: Using Fake Timers
+---
+
+Using real timers in your tests is problematic since they depend on real time
```

Comment:
I feel like the wording here strongly advises the use of fake timers 100% of the time or as the default. I don't know about anyone else, but fake timers as complexity to test code and only sometimes provide any benefit and I therefore only use them when I really want to fake timers (like a debounce). Either way, I don't think this mentions the drawbacks fairly and I think we should do so. I don't think people should come away from reading this with the ideas that fake timers should be their default way for testing. Maybe we could add a section for when fake timers are worth the trouble?

Reply:
Yeah you're right, it's pretty misleading, I'll work on that.

Reply:
Sorry it took me some time, I rephrased it. 
I think it's still not 100% explanatory though, wasn't really sure how I can expand it..